src/components/Modal.js

```diff
@@ -1,24 +1,76 @@
 import Grid from '@mui/material/Grid';
-import Button from '@mui/material/Button';
-import NoteAddIcon from '@mui/icons-material/NoteAdd';
+import { styled } from '@mui/material/styles';
+import Dialog from '@mui/material/Dialog';
+import PropTypes from 'prop-types';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+import DialogTitle from '@mui/material/DialogTitle';
+import AddEditForm from './ManageNoteForm'
 
-export default function CustomizedDialogs({children,setOpen}) {
-  const handleClickOpen = () => {
-    setOpen(true);
-  };
-  const handleClose = () => {
-    setOpen(false);
-  };  
+const BootstrapDialog = styled(Dialog)(({ theme }) => ({
+  '& .MuiDialogContent-root': {
+    padding: theme.spacing(2)
+  },
+  '& .MuiDialogActions-root': {
+    padding: theme.spacing(1),
+  },
+}));
+const BootstrapDialogTitle = ({ children, onClose, setNewForm, setIdEditNote, ...other }) => {
```

Comment:
Again we have two components in the same file, please move it to the separated file